### PR TITLE
Images load without Js

### PR DESCRIPTION
**Pull request by Vishwaak**
Issue Id: #188

Problem: The Images were not displayed with Javascript disabled on the client browser.

How-to-check: Disable Javascript in your browser to see the image Load

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -132,6 +135,9 @@ <h3 class="green-text text-darken-3"><i>An open source voxel world - imagine the
         </div>
         <div class="parallax-container">
             <div class="parallax"><img src="{{ site.baseurl }}/img/ss_2.jpg"></div>
+              </<noscript>
```
Comment (pollend) [inline index.html]: you have an extra un-closed tag here.

**Review by pollend — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Vishwaak) [reply index.html]: It's been resolved.

File: index.html [hunk 2]
```diff
@@ -42,6 +42,9 @@
     <main>
         <div class="parallax-container">
             <div class="parallax"><img src="{{ site.baseurl }}/img/ss_6.jpg"></div>
+              </<noscript>
```
Comment (smsunarto) [inline index.html]: This doesn't look right. Isn't it supposed to be `<noscript>`? and please indent the next line accordingly

**Review by smsunarto — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Vishwaak) [reply index.html]: Sorry about that. I guess this should resolve the issue.

**Comment by Vishwaak** [comment]
All the changes are made. Hoping to get reviewed.

**Comment by Cervator** [comment]
Looks good and is working now on https://terasology.org/ - thanks @Vishwaak ! :-)

**Pull request merged**